onnxruntime/test/providers/cpu/generator/random_test.cc

```diff
@@ -33,7 +33,7 @@ TEST(Random, RandomNormal2DDouble) {
                 [&generator, &distribution](double& value) { value = distribution(generator); });
 
   test.AddOutput<double>("Y", dims, expected_output);
-  test.Run();
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaExecutionProvider});
```

Comment:
I am curious, why these test are not passing for CUDA EP?

Reply:
I used std lib to generate the expected output, which is used for CPU kernel only.

  std::default_random_engine generator{gsl::narrow_cast<uint32_t>(seed)};
  std::normal_distribution<double> distribution{mean, scale};

Reply:
got it. let's add this reason as a comment.